homeassistant/components/flipr/strings.json

```diff
@@ -39,6 +39,11 @@
       "red_ox": {
         "name": "Red OX"
       }
+    },
+    "switch": {
+      "hub_state": {
+        "name": "state"
+      }
```

Comment:
What does the switch mean? If the hub is on or off? If so, I think setting the name to `None` is more appropriate as its the main feature of the hub as it seems. It will then follow the device name

Reply:
I prefer to have the entity to be : switch.flipr_hub_ca6268_state instead of switch.flipr_hub_ca6268_none and a displayed name to the user containing the None word like in "Flipr hub CA6268 None"

Reply:
The fliprhub is not a hub in the sense it manages other devices.
Fliprhub is a device containing a relay (so the state on/off for the relay which is the state) and a mode (manual, auto or planning). In auto or planning mode, the relay is managed via the cloud provider and in manual mode, the relay is managed manually by the flipr_api to switch on or off. This is here that all the automations of home assistant will come into play to set the fliprhub in manual mode and switch the fliprhub on or off.

Reply:
It won't show `None`. It would be `switch.flipr_hub_ca6268` and `Flipr hub CA6268`

Reply:
I found the reason : it was in entity.py, I have to correct it.
It is ok now : no more need for translation_key and the entity name is switch.flipr_hub_ca6268.